client/blocks/upload-image/docs/example.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { partial } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import UploadImage from '../';
+import { AspectRatios } from 'state/ui/editor/image-editor/constants';
+
+export default class UploadImageExample extends Component {
+	state = {
+		isUploading: false,
+		uploadedImageDataUrl: null
```

Comment:
Yeah, I wanted to leave it there as an example for other people to see how to use it. Same goes for your other comment. :)